src/app/home/form/form.component.html

```diff
@@ -0,0 +1,56 @@
+<div>
+  <div class="{{ formClass }}">
+    <div class="card"></div>
+    <div class="card">
+      <h1 class="title">Login</h1>
+      <form>
+        <div class="input-container">
+          <input type="text" id="#{label}" required="required" />
+          <label for="#{label}">Username</label>
+          <div class="bar"></div>
+        </div>
```

Comment:
why using `#{label}` for id? Are we trying to interpolate some values here?

Reply:
will make it basic